guides/release/getting-started/index.md

```diff
@@ -1,7 +1,8 @@
 ## What is Ember?
 
-Ember is a JavaScript front-end framework designed to help you build websites with rich and complex user interactions.
-It does so by providing developers both with many features that are essential to manage complexity in modern web applications,
+Ember.js is a productive, battle-tested JavaScript framework for building modern web applications.
+It includes everything you need to build rich UIs that work on any device. It does so by providing
+developers both with many features that are essential to manage complexity in modern web applications,
```

Comment:
@hergaiety Let me push a commit that removes "both with." I didn't think it sounded right because "as well as" appears later.